spot_rl_experiments/spot_rl/envs/nav_env.py

```diff
@@ -55,6 +55,23 @@ def parse_arguments(args=sys.argv[1:]):
     return args
 
 
+def construct_config_for_nav(file_path=None, opts=[]):
+    """
+    Constructs and updates the config for nav
+
+    Args:
+        file_path (str): Path to the config file
+        opts (list): List of options to update the config
+
+    Returns:
+        config (Config): Updated config object
+    """
+    config = construct_config(file_path=file_path, opts=opts)
```

Comment:
curious why this function is only for nav? Doesn't seem to have any nav specific properties to it? Is it the setting USE_MRCNN to false?

Reply:
The idea for this function is to alter all the modifications needed to the config.yaml specific to navigation tasks